CONTRIBUTING.md

```diff
@@ -83,4 +83,21 @@ This should launch a Koop server on port 8080. You can verify this via the follo
 - http://localhost:8080/marklogic/rest/services/GDeltExample/FeatureServer/0 = should return a JSON feature service 
   descriptor
 
+## Testing the documentation locally
 
+The docs for this project are stored in the `./docs` directory as a set of Markdown files. These are published via
+[GitHub Pages](https://docs.github.com/en/pages/getting-started-with-github-pages/about-github-pages) using the
+configuration found under "Settings / Pages" in this repository. 
+
+You can build and test the docs locally by 
+[following these GitHub instructions](https://docs.github. 
```

Comment:
There's a space after "docs.github." that causes the link to not work.

Reply:
Thanks for catching this - I have word wrap turned on in Intellij, which is helpful for markdown writing, except when this happens.